tests/integration_tests/instances.py

```diff
@@ -43,21 +52,39 @@ def emit_settings_to_log(self) -> None:
     def destroy(self):
         self.instance.delete()
 
-    def execute(self, command):
-        return self.instance.execute(command)
+    def execute(self, command, *, use_sudo=None) -> Result:
```

Comment:
What is '*' for ? you dn't use it anywhere.

Reply:
Designating keyword only arguments. https://www.python.org/dev/peps/pep-3102/
You can't pass it as a positional arg like this
```python
instance.execute('cat stuff', True)
```
You have to do
```python
instance.execute('cat stuff', use_sudo=True)
```